core/drivers/ffa_console.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
"All rights reserved" is useless, see https://en.wikipedia.org/wiki/All_rights_reserved#Obsolescence

Reply:
Done